pkg/web/apps/apps_psql_repository.go

```diff
@@ -15,6 +14,12 @@ type (
 	}
 
 	// PostgreSQLRepository interface defines the methods to be implemented
+	PostgreSQLRepository interface {
```

Comment:
This does not need to be here as the new instance (line 29) uses the `Repository` defined in the same package which is the generic apps_repository

Reply:
The Mock uses the interface